connect/mirror/src/test/java/org/apache/kafka/connect/mirror/MirrorConnectorsIntegrationTest.java

```diff
@@ -1,595 +0,0 @@
-        assertEquals("Did not calculate replication hops correctly.", 1, backupClient.replicationHops("primary"));
-        assertTrue("Did not find upstream backup cluster.", primaryClient.upstreamClusters().contains("backup"));
-        assertEquals("Did not calculate replication hops correctly.", 1, primaryClient.replicationHops("backup"));
-
-        assertTrue("Checkpoints were not emitted downstream to backup cluster.",
-            backup.kafka().consume(1, CHECKPOINT_DURATION_MS, "primary.checkpoints.internal").count() > 0);
-
-        Map<TopicPartition, OffsetAndMetadata> backupOffsets = backupClient.remoteConsumerOffsets(consumerGroupName, "primary",
-            Duration.ofMillis(CHECKPOINT_DURATION_MS));
-
-        assertTrue("Offsets not translated downstream to backup cluster. Found: " + backupOffsets, backupOffsets.containsKey(
-            new TopicPartition("primary.test-topic-1", 0)));
-
-        // Failover consumer group to backup cluster.
-        backupConsumer = backup.kafka().createConsumer(consumerProps);
-        backupConsumer.assign(allPartitions("test-topic-1", "primary.test-topic-1"));
-        seek(backupConsumer, backupOffsets);
-        consumeAllMessages(backupConsumer, 0);
-
-        assertTrue("Consumer failedover to zero offset.", backupConsumer.position(new TopicPartition("primary.test-topic-1", 0)) > 0);
-        assertTrue("Consumer failedover beyond expected offset.", backupConsumer.position(
-            new TopicPartition("primary.test-topic-1", 0)) <= NUM_RECORDS_PRODUCED);
-        assertTrue("Checkpoints were not emitted upstream to primary cluster.", primary.kafka().consume(1,
-            CHECKPOINT_DURATION_MS, "backup.checkpoints.internal").count() > 0);
-
-        backupConsumer.close();
-
-        waitForCondition(() -> {
-            try {
-                return primaryClient.remoteConsumerOffsets(consumerGroupName, "backup",
-                    Duration.ofMillis(CHECKPOINT_DURATION_MS)).containsKey(new TopicPartition("backup.test-topic-1", 0));
-            } catch (Throwable e) {
-                return false;
-            }
-        }, CHECKPOINT_DURATION_MS, "Offsets not translated downstream to primary cluster.");
-
-        waitForCondition(() -> {
-            try {
-                return primaryClient.remoteConsumerOffsets(consumerGroupName, "backup",
-                    Duration.ofMillis(CHECKPOINT_DURATION_MS)).containsKey(new TopicPartition("test-topic-1", 0));
-            } catch (Throwable e) {
-                return false;
-            }
-        }, CHECKPOINT_DURATION_MS, "Offsets not translated upstream to primary cluster.");
-
-        Map<TopicPartition, OffsetAndMetadata> primaryOffsets = primaryClient.remoteConsumerOffsets(consumerGroupName, "backup",
-                Duration.ofMillis(CHECKPOINT_DURATION_MS));
-
-        primaryClient.close();
-        backupClient.close();
```

Comment:
It looks like we lost these lines and maybe some others too. These came from https://github.com/apache/kafka/pull/9698. Can you make sure we don't miss any existing logic?